evm/src/executor/inspector/cheatcodes/util.rs

```diff
@@ -151,6 +151,10 @@ pub fn apply<DB: Database>(
             state.labels.insert(inner.0, inner.1.clone());
             Ok(Bytes::new())
         }
+        HEVMCalls::GetLabel(inner) => {
+            let label = String::from(state.labels.get(&inner.0).or(Some(&String::from(format!("unlabeled:{}", inner.0)))).unwrap());
+            Ok(Bytes::from(ethers::abi::encode(&[Token::String(label)])))
```

Comment:
1. `Type::from(x)` -> `x.into()`, unless it can't be inferred
    - also `format!()` returns `String`, so this can be removed entirely

2. `option.or(Some(x)).unwrap()` -> `option.unwrap_or(x)` -> `option.unwrap_or_else(|| x)`
    - the last step is making it lazy since it's a function call, see [`Option::unwrap_or` docs](https://doc.rust-lang.org/stable/std/option/enum.Option.html#method.unwrap_or)

```suggestion
            let label = state.labels.get(&inner.0).cloned().unwrap_or_else(|| format!("unlabeled:{}", inner.0));
            Ok(ethers::abi::encode(&[Token::String(label)]).into())
```